r/tools/nixlibs.R

```diff
@@ -222,7 +222,7 @@ check_allowlist <- function(os, allowed = "https://raw.githubusercontent.com/apa
     # Try a remote allowlist so that we can add/remove without a release
     suppressWarnings(readLines(allowed)),
     # Fallback to default: allowed only on Ubuntu and CentOS/RHEL
-    error = function(e) c("ubuntu", "centos", "redhat", "rhel")
+    error = function(e) c("ubuntu", "centos", "redhat", "rhel", "rocky", "opensuse-leap")
```

Comment:
I don't recall the history on this, but the local copy of the allowlist should be present here, right? Can we fall back to `readLines("nixlibs-allowlist.txt")` (or whatever path it is relative to cwd)? That way we only have to track this in one place.

Also we should update the comment on the line above.

Reply:
Ah, of course